### DISCUSSION TITLE
Other public entities are directly involved in the e-procurement phases

### DISCUSSION THREAD

**Discussion by eprocurementontology** [eprocurementontology/eprocurementontology#507] [Ideas]
Jáchym Hercher submitted the following use case to be further developed:
Other public entities are directly involved in the e-procurement phases described in the Project Charter. This includes:

1. Creating new information (e.g. review authority freezing the procurement process, rejecting a complaint, or awarding damages).
2. Exchanging data between e-procurement systems and systems used by auditors and review bodies, so that it is easier for them to check the validity of the procurement process.

**Comment by muricna** [comment]
I am not sure I understand the first point.

**Comment by JachymHercher** [comment]
For example, when the review body decides that "We have decided that the procurement procedure should be frozen until we review complaint ABC123", then this decision is newly created information. 

The point I was trying to make was to distinguish it from information which gets taken over from other databases (e.g. "Reviewed procedure has a monetary value of EUR 10000" which is information used by the review body but not created by the review body.).
